src/script.js

```diff
@@ -163,53 +163,41 @@ function getUserInputValues() {
         // Pridobi vnose energij
         let e_cbx = parseFloat(document.getElementById(`e-cb${i}`).value);
         if (isNaN(e_cbx) || e_cbx < 0)    // Preveri, če je uporabnik pustil polje prazno ali če je vrednost negativna
-        {e_cb.push(0);}
-        else
-        {e_cb.push(e_cbx);}
+        { e_cb.push(0); }
+        else { e_cb.push(e_cbx); }
         // Pridobi vnose dog. moči
         let m_cbx = parseFloat(document.getElementById(`m-cb${i}`).value);
-        if (isNaN(m_cbx) || m_cbx < 0)
-        {m_cb.push(0);}
-        else
-        {m_cb.push(m_cbx);}
+        if (isNaN(m_cbx) || m_cbx < 0) { m_cb.push(0); }
+        else { m_cb.push(m_cbx); }
         // Pridobi vnose presežkov
         let m_ex_cbx = parseFloat(document.getElementById(`m-ex-cb${i}`).value);
-        if (isNaN(m_ex_cbx) || m_ex_cbx < 0)
-        {m_ex_cb.push(0);}
-        else
-        {m_ex_cb.push(m_ex_cbx);}
+        if (isNaN(m_ex_cbx) || m_ex_cbx < 0) { m_ex_cb.push(0); }
+        else { m_ex_cb.push(m_ex_cbx); }
     }
-    
+
     // Pridobi vnos uporabnika - porabljena energija in dogovorjene moči po tarifah
     // Pridobi vnos energije MT
     let e_mt = parseFloat(document.getElementById('e-mt').value);
-    if (isNaN(e_mt) || e_mt < 0)
-    {e_mt = 0;}
+    if (isNaN(e_mt) || e_mt < 0) { e_mt = 0; }
     // Pridobi vnos energije VT
     let e_vt = parseFloat(document.getElementById('e-vt').value);
-    if (isNaN(e_vt) || e_vt < 0)
-    {e_vt = 0;}
+    if (isNaN(e_vt) || e_vt < 0) { e_vt = 0; }
     // Pridobi vnos energije ET
     let e_et = parseFloat(document.getElementById('e-et').value);
-    if (isNaN(e_et) || e_et < 0)
-    {e_et = 0;}
+    if (isNaN(e_et) || e_et < 0) { e_et = 0; }
     // Pridobi vnos obračunske moči
     let m_obr = parseFloat(document.getElementById('m-obr').value);
-    if (isNaN(m_obr) || m_obr < 0)
-    {m_obr = 0;}
+    if (isNaN(m_obr) || m_obr < 0) { m_obr = 0; }
 
     // Pridobi vnos uporabnika - cena energenta
     let price_en_mt = parseFloat(document.getElementById('price-en-mt').value);
-    if (Number.isNaN(price_en_mt))
-    {price_en_mt = 0;}
+    if (Number.isNaN(price_en_mt)) { price_en_mt = 0; }
 
     let price_en_vt = parseFloat(document.getElementById('price-en-vt').value);
-    if (Number.isNaN(price_en_vt))
-    {price_en_vt = 0;}
-    
+    if (Number.isNaN(price_en_vt)) { price_en_vt = 0; }
+
     let price_en_et = parseFloat(document.getElementById('price-en-et').value);
-    if (Number.isNaN(price_en_et))
-    {price_en_et = 0;}
+    if (Number.isNaN(price_en_et)) { price_en_et = 0; }
```

Comment:
if you write the same thing twice then would be better to use function try to not reapeat your self.
function parseInputValue(elementId) {
    let value = parseFloat(document.getElementById(elementId).value);
    return !isNaN(value) && value >= 0 ? value : 0;
}

// Usage example:
let e_mt = parseInputValue('e-mt');
let e_vt = parseInputValue('e-vt');
let e_et = parseInputValue('e-et');
let m_obr = parseInputValue('m-obr');
let price_en_mt = parseInputValue('price-en-mt');
let price_en_vt = parseInputValue('price-en-vt');
let price_en_et = parseInputValue('price-en-et');